readme.md

```diff
@@ -654,7 +655,6 @@ Make sure to see [contributing.md](/contributing.md) for instructions on contrib
 * [PosXML](p/PosXML.posxml)
 * [Potaka](p/Potaka.ptk)
 * [Potigol](p/Potigol.poti)
-* [PowerBuilder](p/PowerBuilder.psr)
```

Comment:
This shouldn't be removed

Reply:
oh sorry, that wasn't my intention, i'm fixing this, just a minute